From 4053dc46969b56084fb72c24ae062187134cd39a Mon Sep 17 00:00:00 2001
From: David Kalnischkies <kalnischkies@gmail.com>
Date: Wed, 18 Apr 2012 17:18:40 +0200
Subject: [PATCH] directly include declaration of MD5Summation

While working on apt 0.9 I cleaned up the header includes. I thought
I had guarded these changes with APT_8_CLEANER_HEADERS, but it seems
I forgot one, which makes qapt FTBFS as it depends on such a transitive
include. As it is a good idea to avoid these transitive includes anyway
lets do the include in qapt instead of readding it to apt.
---
 src/debfile.cpp |    1 +
 1 file changed, 1 insertion(+)

diff --git a/src/debfile.cpp b/src/debfile.cpp
index c315f53..2f93afd 100644
--- a/src/debfile.cpp
+++ b/src/debfile.cpp
@@ -28,6 +28,7 @@
 #include <apt-pkg/debfile.h>
 #include <apt-pkg/fileutl.h>
 #include <apt-pkg/tagfile.h>
+#include <apt-pkg/md5.h>
 
 #include <QDebug>
 
-- 
1.7.9.5

